community.html

```diff
@@ -71,6 +71,19 @@ <h2><i class="fa fa-twitter"></i> Scrapy tweets</h2>
     </p>
   </div>
 
+  <div class="community-box">
+    <a href="https://t.me/scrapy_python">
+      <h2><i class="fa fa-paper-plane"></i>Telegram community</h2>
```

Comment:
you can use `fa-telegram` class instead of `fa-paper-plan`
https://fontawesome.com/v4.7.0/icon/telegram

Reply:
Thank you for the review. Looks like we use 4.6.3. Need to update it?
https://github.com/scrapy/scrapy.org/blob/master/css/font-awesome.css